src/components/Landing/WelcomeModule/WelcomeModule.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './WelcomeModule.module.scss';
+
+const WelcomeModule = () => (
+  <div className={styles['welcome__module']}>
+    <img
+      className={styles['welcome__module__image']}
+      src='/workspace.png'
+    ></img>
```

Comment:
Upgrade the image implementation using Gatsby Image. Don't forget to add the following attributes: `alt`, `width` and `height`.